themes-default/slim/views/config_postProcessing.mako

```diff
@@ -290,16 +309,16 @@ const startVue = () => {
                         <fieldset class="component-group-list">
 
                             <!-- default name-pattern component -->
-                            <name-pattern :pattern="pattern" :presets="presets"></name-pattern>
+                            <name-pattern :enabled="true" :naming-pattern="pattern" :naming-presets="presets" :multi-ep-style="multiEpSelected" :multi-ep-styles="multiEpStrings" @change="onChangePattern" ></name-pattern>
```

Comment:
Since this is the default why not move this data to the components data object as default values?